frontend/src/components/LevelPage/GameMenu/Objectives/Objectives.jsx

```diff
@@ -23,14 +25,28 @@ function Objectives() {
           showObjectives ? "objectivesPopUp__show" : ""
         }`}
       >
-        <button
-          type="button"
-          className="objectivesPopUp__closeButton"
-          aria-label="Fermer la pop-up"
-          onClick={handleObjectivesToggle}
-        >
-          X
-        </button>
+        <header className="headerObjectives">
+          <button
+            type="button"
+            className="objectivesPopUp__closeButton"
+            onClick={handleObjectivesToggle}
+          >
+            X
+          </button>
+          <h2 className="objectivesTitle">Objectifs</h2>
+        </header>
+        <ul className="objectivesList">
+          {objectives[0].map((e) => (
```

Comment:
I don't understand the logic behind it

Reply:
We need objectives to be showed on the notebook, i add to each other a class that crossed text when the item is in bag, objectives[0] is an array of objects witch contain objectives and item attached to the objective, to check when this item is in bag.